core/src/main/scala/tech/beshu/ror/accesscontrol/factory/decoders/rules/LdapRulesDecoders.scala

```diff
@@ -110,30 +112,40 @@ class LdapAuthorizationRuleDecoder(ldapDefinitions: Definitions[LdapService],
 }
 
 object LdapAuthorizationRuleDecoder {
+  def ERROR_MSG_NO_GROUPS_LIST(name: String) = s"Please specify one between 'groups' or 'groups_and' for LDAP authorization rule '${name}'"
```

Comment:
1. we don't use the Screaming Snake Case convention in our codebase (in scala in general too)
2. IMO, it's not a good practice to externalize messages like this only because we'd like to reuse them in tests. In tests, the strings/messages should be hardcoded

Reply:
no commented, not fixed

Reply:
oh shoot. I forgot this!